README.md

```diff
@@ -1,6 +1,9 @@
 # Generate Navigational Components With Ease !!!
-[![NPM version][npm-image]][npm-url] [![Build Status][travis-image]][travis-url] [![Dependency Status][daviddm-image]][daviddm-url] [![Coverage Status][coveralls-image]][coveralls-url]
 
+[![NPM version][npm-image]][npm-url]
```

Comment:
does this display the badges one below the other? or in a horizontal line?
I would like to keep it on a single line.
So if this doesn't do that please revert this change.

Reply:
The badges keep to be displayed in horizontal line. The README.md does not displayed white line for badges. I have formatted it only for more visibility of the lines